test/pxd_fastpath_test.cc

```diff
@@ -0,0 +1,927 @@
+#include <algorithm>
+#include <boost/iostreams/device/file_descriptor.hpp>
+#include <boost/lexical_cast.hpp>
+#include <fcntl.h>
+#include <fstream>
+#include <functional>
+#include <gtest/gtest.h>
+#include <stdlib.h>
+#include <string>
+#include <sys/ioctl.h>
+#include <sys/poll.h>
+#include <sys/stat.h>
+#include <sys/uio.h>
+#include <sys/mount.h>
+#include <thread>
+#include <vector>
+#include <unistd.h>
+#include <linux/fs.h>
+#include <set>
+
+#include <cstdlib>
+#include <memory>
+#include <stdexcept>
+
+#include "fuse.h"
+#include "pxd.h"
+
+using namespace std::placeholders;
+
+// Helper function to create temporary backing files for fastpath
+class TempBackingFile {
```

Comment:
this is file backed fastpath, you need to create block devices for fastpath instead to do block based fatpath.
This code path is never run in production.

Reply:
you can keep this and add a loopdevice backed test for block device mapped fastpath tests.